joint_limits:

  joint4_position_controller:

    has_position_limits: true
    min_position: -0.48
    max_position: 0.48
    has_velocity_limits: true
    max_velocity: 0.0
    has_acceleration_limits: true
    max_acceleration: 0.0
    has_jerk_limits: true
    max_jerk: 0.0
    has_effort_limits: true
    max_effort: 1.0

  joint5_position_controller:

    has_position_limits: true
    min_position: -0.48
    max_position: 0.48
    has_velocity_limits: true
    max_velocity: 0.0
    has_acceleration_limits: true
    max_acceleration: 0.0
    has_jerk_limits: true
    max_jerk: 0.0
    has_effort_limits: true
    max_effort: 1.0
